fmf/utils.py

```diff
@@ -236,6 +244,8 @@ def filter(filter, data, sensitive=True, regexp=False):
     filter is not found in the data dictionary. Set option 'sensitive'
     to False to enable case-insensitive matching. If 'regexp' option is
     True, regular expressions can be used in the filter values as well.
+    However, using either '|' or '&' in the regular expression is not
+    allowed.
```

Comment:
So I guess, the `|` could/should be covered by #220 which should be included in the `1.4` release as well. Shall we extend the pull request to support `&` escaping as well?